src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid $card-border-color;
+  color: $card-font-color;
+  padding: 32px $card-horiz-margin 0;
+
+  &__img {
+    display: block;
+    width: $img-width;
+    height: $img-height;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: $card-title-size;
+    line-height: $product-title-line-height;
+    font-weight: $card-title-weight;
+  }
+
+  &__code {
+    font-size: $product-code-size;
+    color: $product-code-color;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-top: 24px;
+  }
+
+  &__price-title {
+    margin: 0;
+    font-size: 12px;
+    color: $product-code-color;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    text-decoration: none;
+    height: $cta-height;
+    background-color: $cta-color;
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    line-height: $cta-line-height;
+    color: $cta-font-color;
+    font-size: $cta-font-size;
+    line-height: $cta-line-height;
+    font-weight: $cta-font-weight;
+    border-radius: $cta-border-radius;
+    margin: 16px 0;
+    text-transform: uppercase;
+
+    &:hover {
+      border: 1px solid #00acdc;
+      background-color: white;
+      color: $cta-color;
+    }
```

Comment:
For the ':hover' pseudo-class, consider using SCSS variables for colors (e.g., for border color) to maintain consistency and ease of maintenance, as is done with other color properties in this file.